projectRM/projectRM/Views/ProductViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ViewController.swift
+//  projectRM
+//
+//  Created by Александр Троицкий on 21.09.2022.
+//
+
+import UIKit
+/// Экран, который позволяет выбрать себе модель телефона через сегмент контрол для покупки и перехода на экран дополнительных условий покупки
+class ProductViewController: UIViewController {
+
+    var nameOfProductLabel = UILabel()
+    var segmentControl = UISegmentedControl()
+    var menuArray = ["iPhone 14", "iPhone 12", "iPhone 13"]
+    var imageView = UIImageView()
+    var selectButton = UIButton()
+    
+    var imageArray = [UIImage(named: "iphoneOne.jpeg"),
```

Comment:
```suggestion
    var images = [UIImage(named: "iphoneOne.jpeg"),
```
Забываем слово array